README.md

```diff
@@ -69,23 +70,51 @@ _ = refl
 ```
 
 Yeah, that was easier… However, the SMT solver also allows us to run universally
-and existentially quantified expressions:
+and existentially quantified expressions. For example we might want to the network
+models the AND gate correctly for any values sufficiently close to 0.0 and 1.0.
+This can be done by defining some new predicates `truthy` and `falsey`:
+
+```agda
+ε : ∀ {Γ} → Term Γ REAL
+ε = toReal 0.2
+
+truthy : ∀ {Γ} → Term Γ REAL → Term Γ BOOL
+truthy x = ∣ 1·0f - x ∣ ≤ ε
+
+falsey : ∀ {Γ} → Term Γ REAL → Term Γ BOOL
+falsey x = ∣ 0·0f - x ∣ ≤ ε
+```
+
+We can then rewrite our constraints to use these fuzzy values:
+
+```agda
+fuzzyConstraints : NetworkConstraints 2 1
+fuzzyConstraints (i₀ ∷ i₁ ∷ []) (o₀ ∷ []) =
+  (falsey i₀ ∧ falsey i₁ ⇒ falsey o₀) ∷
+  (falsey i₀ ∧ truthy i₁ ⇒ falsey o₀) ∷
+  (truthy i₀ ∧ falsey i₁ ⇒ falsey o₀) ∷
+  (truthy i₀ ∧ truthy i₁ ⇒ truthy o₀) ∷
+  []
+```
+
+and then rerun the query:
 
 ```agda
--- pending…
+_ : z3 (query model fuzzyConstraints) ≡ unsat ∷ []
+_ = refl
 ```
 
-Whoa, that’s very cool! Now we know our network is robust around both truthy and
+Whoa, that’s very cool! Now we know our network is robust around both truthy and
```

Comment:
I'm very confused as to why this particular line was picked up by diff?

Reply:
I think it was a non-standard white space character before. Emacs was highlighting it at any rate...

Reply:
Wonder how I managed to insert that...